modules/js/planetaryFinal.js

```diff
@@ -76,129 +101,114 @@ function addPlanetaryGearComposite(centerX, centerY, constraintX, constraintY){
-  removeComposite(compositeArray[1].bodies[0])
-  addPlanetaryGearComposite(compositeArray[0].constraints[0].pointA.x - compositeArray[0].radius - radius - ((toothHeight*0.6)*2), (window.innerHeight)*(0.65),compositeArray[0].constraints[0].pointA.x,compositeArray[0].constraints[0].pointA.y);
+  addPlanetaryGearComposite(
+    compositeArray[0].constraints[0].pointA.x -
+      compositeArray[0].radius -
+      radius -
+      toothHeight * 0.6 * 2,
+    window.innerHeight * 0.65,
+    compositeArray[0].constraints[0].pointA.x,
+    compositeArray[0].constraints[0].pointA.y
+  );
 }
-// see smallGear1()
-function largeGear1(){
-  Body.setAngle(compositeArray[0].bodies[0], 0)
-  Body.setAngle(compositeArray[1].bodies[0], 0)
-  radius = 80;
-  compositeArray[1].radius = radius
-  steps = (0.25 * radius)*2;
-  toothWidthDegree = 2;
-  toothWidth = (toothWidthDegree/conversionFactor);
-  removeComposite(compositeArray[1].bodies[0])
-  addPlanetaryGearComposite(compositeArray[0].constraints[0].pointA.x - compositeArray[0].radius - radius - ((toothHeight*0.6)*2), (window.innerHeight)*(0.65),compositeArray[0].constraints[0].pointA.x,compositeArray[0].constraints[0].pointA.y);
-}
-// see smallGear1()
-function smallGear2(){
-  Body.setAngle(compositeArray[0].bodies[0], 0)
-  Body.setAngle(compositeArray[1].bodies[0], 0)
-  radius = 48;
-  compositeArray[0].radius = radius
-  steps = (0.25 * radius)*2;
-  toothWidthDegree = 4;
-  toothWidth = (toothWidthDegree/conversionFactor);
-  changeBodyPlanetary(0);
-}
-// see smallGear1()
-function mediumGear2(){
-  Body.setAngle(compositeArray[0].bodies[0], 0)
-  Body.setAngle(compositeArray[1].bodies[0], 0)
-  radius = 64;
-  compositeArray[0].radius = radius
-  steps = (0.25 * radius)*2;
-  toothWidthDegree = 3;
-  toothWidth = (toothWidthDegree/conversionFactor);
-  changeBodyPlanetary(0);
-}
-// see smallGear1()
-function largeGear2(){
-  Body.setAngle(compositeArray[0].bodies[0], 0)
-  Body.setAngle(compositeArray[1].bodies[0], 0)
-  radius = 80;
+
+function gear2(rad) {
+  Body.setAngle(compositeArray[0].bodies[0], 0);
+  Body.setAngle(compositeArray[1].bodies[0], 0);
+  radius = rad;
   compositeArray[0].radius = radius;
-  steps = (0.25 * radius)*2;
-  toothWidthDegree = 2;
-  toothWidth = (toothWidthDegree/conversionFactor);
+  steps = 0.25 * radius * 2;
+  toothWidthDegree = 4;
+  toothWidth = toothWidthDegree / conversionFactor;
   changeBodyPlanetary(0);
 }
 ////////////////////////////////////////////////////////////////////////////////////////////////
-Events.on(engine, 'beforeUpdate', function(event) {
-  Body.setPosition(compositeArray[1].bodies[1],compositeArray[1].bodies[0].position)
-  for(var i = 0; i<compositeArray.length; i++){
-    if(compositeArray[i].constraints[0]){
+Events.on(engine, "beforeUpdate", function (event) {
+  Body.setPosition(
+    compositeArray[1].bodies[1],
+    compositeArray[1].bodies[0].position
+  );
+  for (let i = 0; i < compositeArray.length; i++) {
```

Comment:
I know this repo leans heavily on `for` loops.  This would be a good place for a `forEach` loop:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach